src/legacy/core_plugins/data/public/search/search_bar/components/saved_query_management/saved_query_management_component.tsx

```diff
@@ -61,18 +60,26 @@ export const SavedQueryManagementComponent: FunctionComponent<Props> = ({
 }) => {
   const [isOpen, setIsOpen] = useState(false);
   const [savedQueries, setSavedQueries] = useState([] as SavedQuery[]);
+  const [count, setTotalCount] = useState(0);
   const [activePage, setActivePage] = useState(0);
 
   useEffect(() => {
-    const fetchQueries = async () => {
-      const allSavedQueries = await savedQueryService.getAllSavedQueries();
-      const sortedAllSavedQueries = sortBy(allSavedQueries, 'attributes.title');
-      setSavedQueries(sortedAllSavedQueries);
+    const fetchCountAndSavedQueries = async () => {
+      const savedQueryCount = await savedQueryService.getSavedQueryCount();
+      setTotalCount(savedQueryCount);
+
+      const savedQueryItems = await savedQueryService.findSavedQueries(
+        undefined,
```

Comment:
Will this work instead? 

```suggestion
        '',
```

Reply:
https://github.com/elastic/kibana/pull/47646/commits/a4b4746f79aa24fa593c0ddc776eb36a98d15ca3#diff-83cc6f50604f418b6c214b7989046d90R71